lib/ui/painting.dart

```diff
@@ -1520,6 +1520,11 @@ class _ImageInfo {
 ///
 /// To draw an [Image], use one of the methods on the [Canvas] class, such as
 /// [Canvas.drawImage].
+///
+/// See also:
+///
+///  * [Image], the class in the [widgets] library that encapsulates this class.
```

Comment:
Does just `[Image]` work? How does DartDocs know to link to the one from flutter/flutter and not just link to Image in the same file here?

Reply:
According to the docs, `[Image]` resolves to the higher layer, while `[dart:ui.Image]` resolves to the lower layer.